### PR TITLE
Apply margin bottom helper to big number component

### PR DESCRIPTION
**Pull request by owenatgov**
## What
Applies the shared helper, which includes standardised `margin_bottom` behaviour, to [the big number component](https://components.publishing.service.gov.uk/component-guide/big_number).

## Why
Fixes https://github.com/alphagov/govuk_publishing_components/issues/2479

[Card](https://trello.com/c/u0DGQWQY/724-add-margin-bottom-to-big-number-component-and-apply-to-the-homepage)

### New docs section
Screenshot 2022-01-10 at 17 34 23

### CODE REVIEW

**Comment by owenatgov** [comment]
Just spotted that this needs a test for the new attribute. I initially thought that this wouldn't need one as this relies on an external helper which as its own tests but other components using the helper add tests for `margin_bottom`. Will follow convention here.

**Comment by andysellick** [comment]
@owenatgov good shout. Yes, the tests don't need to be exhaustive because as you pointed out this functionality already has tests, but a basic test to check that the margin helper has been implemented correctly on this component would be good 👍

File: spec/components/big_number_spec.rb
Comment (andysellick) [inline spec/components/big_number_spec.rb:84]: Nitpick: typo: `bottm`.

**Review by andysellick — APPROVED** [review]
One tiny typo, but otherwise looks good 👍

**Comment by owenatgov** [comment]
@andysellick Would you mind approving Percy? Once you do (if you're happy with it) I'll merge this.

**Pull request merged**
